### ISSUE TITLE
Lawnchair 2 ideas from @felixhammerl

### ISSUE THREAD

**Issue by brianleroux** [brianleroux/lawnchair#208]
```
Add linting
```

Yes pls!

```
Unify code style (strict mode, indent w/ four spaces, no comma-first, semicolons, curly braches in conditional statements, ...)
```

No. This is _my_ project and it follows _my_ coding style. I respect other projects with pull requests in their style. That's what the linter can do! If anything these days I go: no semicolons, 2 spaces, comma first, single line conditionals. [This is a thought out style not blindly following old advice.](https://docs.npmjs.com/misc/coding-style) 

```
Remove new Function used in the terse callbacks
```

No. Its there for a reason. Before Crockford vilified eval it was very common pattern in JS. Don't beleive hype. It is useful to understand your tools: all of them. 

```
Add continuous integration (e.g. via tastling) for multiple browser configurations
```

Travis CI is here. Testling is no longer maintained, alas.

```
Port the tests to mocha+chai if necessary
```

I'd rather we move to Tape instead. [No globals. No augmented built ins.](https://medium.com/@brianleroux/es6-modules-amd-and-commonjs-c1acefbe6fc0#5aa9-e2049e347327)

```
Port lawnchair to promises (if and where that makes sense) to improve exception handling
```

No. This would require a dependency and the various libs that do this do it in diffferent ways. Then if in userland someone chooses a different lib you run into conflicts. We see it in PhoneGap a lot. Promises can come when they are native to the language. And quite bluntly, they add exactly zero value IMO but they do slow down code and create hard to follow stack traces.

```
Resume proper semantic versioning and publishing to npm, etc.
```

Agree!

```
Remove obsolete incomplete code paths across the modules
```

Agree!

**Comment by felixhammerl** [comment]
Things are going haywire here, so you guys go ahead w/o me for now. I can't promise when I'll have the resources and the time to work on this. F***. :(
